packages/core/plop/stories/index.js

```diff
@@ -1,48 +1,46 @@
-const fs = require("fs");
+import fs from "fs";
```

Comment:
are you sure this works?

Reply:
Yep, tested, and this is how they suggest in their docs (with ESM), why do you ask?

Reply:
I remember I tried to use them with ESM and it didn't worked. maybe relates to the upgraded version

Reply:
Yes it was a bit unintuitive